### PR TITLE
Added enrichment parsers for boolean, integer, and double types

### PR DESCRIPTION
**Pull request by cevaris**
@alexanderdean Let me know what you think and I can patch accordingly. Sort of took this and ran with it.  #35

### CODE REVIEW

**Comment by alexanderdean** [comment]
Wow! This is so cool @cevaris - thanks so much for contributing. Boring admin from me - can you sign our CLA: https://github.com/snowplow/snowplow/wiki/CLA

I will leave feedback/discussion to @chuwy who is our Data Engineering Intern working on this release - over to you Anton!

Thanks again...

File: src/main/scala/com.snowplowanalytics/schemaguru/generators/JsonSchemaGenerator.scala [hunk 1]
```diff
@@ -242,7 +291,12 @@ object JsonSchemaGenerator {
      * @return JsonSchemaType with recognized properties
      */
     def enrichString(value: String) = {
-      guessFormat(value, formatSuggestions) match {
```
Comment (chuwy) [inline src/main/scala/com.snowplowanalytics/schemaguru/generators/JsonSchemaGenerator.scala]: Hi @cevaris Thanks for your interest!
Can you provide me some information about usage of "format" for types other than string and with values like "short" or "boolean". AFAIK we can use just date-time, email, hostname, ipv4, ipv6, uri. [Specification](http://json-schema.org/latest/json-schema-validation.html#anchor104) is uncertain about other types, but I can't see any reason why we should add "format: boolean" to "type: boolean". Some [sources](http://spacetelescope.github.io/understanding-json-schema/reference/string.html) claim "format" is only for strings.

Comment (cevaris) [inline src/main/scala/com.snowplowanalytics/schemaguru/generators/JsonSchemaGenerator.scala]: @chuwy  Sure thing. Strings that are found to be of  `short`, `int`, `long`  are all json `number` json schema type. Thus, the format value can be used for additional schema validations of the `number` json type.

Yeah, I was not going of a json spec. From looking at the code, it seemed the best way to capture specific ranges of numbers.

Comment (chuwy) [inline src/main/scala/com.snowplowanalytics/schemaguru/generators/JsonSchemaGenerator.scala]: @cevaris ok, we'll look if it works for us. Thank you again. Any other contributions would be highly appreciated.

**Pull request closed without merging**